zipline-gradle-plugin/build.gradle.kts

```diff
@@ -44,6 +45,14 @@ gradlePlugin {
   }
 }
 
+pluginManager.withPlugin("org.jetbrains.kotlin.js") {
```

Comment:
This has no effect here. Did you mean to put it in the plugin implementation?

Reply:
Yeah, somebody in #gradle Slack had suggested that block though it looked like gradle.kts and not .kt like would be in the plugin implementation. Would you approach it by adding this configuring to the `override fun apply(project: Project)` so you can then get the tasks through the project?

Reply:
Yep. Should copy/paste right in.

Reply:
Works!